src/SmartComponents/Inventory/applications/advisor/ExpandableRulesCard.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classNames from 'classnames';
+import {
+    ChevronRightIcon,
+    ChevronDownIcon,
+    ThumbsUpIcon,
+    BullseyeIcon,
+    LightbulbIcon
+} from '@patternfly/react-icons';
+
+import { Ansible, Battery } from '../../../../index';
+import { Card, CardBody, CardHeader, Grid, GridItem, List, ListItem, Split, SplitItem } from '@patternfly/react-core';
+
+class ExpandableRulesCard extends React.Component {
+    state = {
+        expanded: true
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.isExpanded !== prevProps.isExpanded) {
+            this.setState({ expanded: this.props.isExpanded });
+        }
+    }
+
+    toggleExpanded = () => {
+        this.setState({ expanded: !this.state.expanded });
+    }
+
+    render() {
+        const { report } = this.props;
+        const rule = report.rule || report;
+        const { expanded } = this.state;
+
+        let rulesCardClasses = classNames(
+            'ins-c-rules-card',
+            'pf-t-light',
+            'pf-m-opaque-100'
+        );
+        return (
+            <Card className={ rulesCardClasses } widget-type='InsightsRulesCard'>
+                <CardHeader>
+                    <Split onClick={ this.toggleExpanded }>
+                        <SplitItem>
+                            { !expanded && <ChevronRightIcon /> } { expanded && <ChevronDownIcon /> }
+                        </SplitItem>
+                        <SplitItem> { rule.category.name } &gt; </SplitItem>
+                        <SplitItem isMain> { rule.description } </SplitItem>
+                        <SplitItem>
+                            <Ansible unsupported={ !rule.has_playbook } />
+                        </SplitItem>
+                    </Split>
+                    <Split>
+                        <SplitItem> <Battery label='Impact' severity={ rule.impact.impact } /> </SplitItem>
+                        <SplitItem> <Battery label='Likelihood' severity={ rule.likelihood } /> </SplitItem>
+                        <SplitItem> <Battery label='Total Risk' severity={ rule.severity } /> </SplitItem>
+                        <SplitItem><Battery label='Risk Of Change' severity={ rule.resolution_risk } /></SplitItem>
+                    </Split>
+                </CardHeader>
+                { expanded && (<CardBody>
+                    <Grid gutter='md' sm={ 12 }>
+                        <GridItem>
+                            <Card className='pf-t-light  pf-m-opaque-100'>
+                                <CardHeader> <ThumbsUpIcon /> Detected Issues</CardHeader>
+                                <CardBody>
+                                    <div dangerouslySetInnerHTML={ { __html: rule.reason_html } } />
```

Comment:
FYI, we plan to remove these _html properties from the rule.  in this case you would just use .reason